index.html

```diff
@@ -184,10 +185,38 @@ <h2 class="connect">
                         <p class="about-p">Skills</p>
                         <i class="fas fa-chevron-right"></i>
                     </div>
-                    
+
+                </div>
+            </section>
+
+            <!-- form section -->
+            <section>
```

Comment:
Please add background images to the form as can be seen on the design page.

Reply:
I've added the background image